idaes/generic_models/properties/core/state_definitions/FTPx.py

```diff
@@ -340,10 +331,11 @@ def state_initialization(b):
              henry_comps,
              l_only_comps,
              v_only_comps) = _valid_VL_component_list(b, pp)
+            pp_VLE = pp
             
-            H = {j: value(get_method(b, "henry_component", j, l_phase)(
-                        b, l_phase, j, b.temperature))
-                        for j in henry_comps}
+            # H = {j: value(get_method(b, "henry_component", j, l_phase)(
```

Comment:
It API for Henry's Law should have stabilized now, so we should look at adding all of this back in.

Core code is in `idaes/generic_models/properties/core/phase_equil/henry.py`
See lines 762-764 in `/idaes/power_generation/carbon_capture/mea_solvent_system/properties/MEA_solvent.py` for an example of the new format.

This lets the user specify the basis for Henry's Law and we currently have support for using either mole fraction or concentration and both "Henry's Constant" and "Henry Volatility".